resources/views/components/news/carousel-item.blade.php

```diff
@@ -22,7 +22,7 @@ class="py-6 px-8 absolute top-0 left-0 w-full h-full"
         </h4>
 
         <p class="newstext shadowoutline lg:opacity-0 transition delay-300 duration-300 subpixel-antialiased {{ $index === 0 ? '!opacity-100' : '' }}">
-            {!! $news->Payload !!}
+            {{ $news->Payload }}
```

Comment:
> The home page news carousel no longer supports HTML of any kind.

Do we need to migrate the existing data? At a minimum, `<br>` should be converted to newline. I'm not sure what to do about links.

![image](https://github.com/user-attachments/assets/6da728b1-2eba-4dbe-9083-a493553cc3b7)

Reply:
A lot of existing data exceeds the 238 character limit.
```
MariaDB [retroachievements]> select count(*) from News where length(Payload) > 238;
+----------+
| count(*) |
+----------+
|     1448 |
+----------+
```

Reply:
We should indeed convert `<br>` to newline - I think all the others we just let go. Currently, news items past 10 are completely inaccessible anyway. When we have a real feed, we should have similar cutoffs (just like the filament panel itself doesn't let the user see posts past a certain date).

Reply:
This change might be jarring for some users. The blurb in the above image effectively just becomes this:
![image](https://github.com/user-attachments/assets/45832594-3911-44b8-8d7b-3c658b3b2134)

The forum link is redundant with the title link, and there's nowhere to actually link to the magazine. Instead, the user has to click the title link (which I admit I never knew was clickable), and then click on the magazine link from there.